charts/latest/csi-driver-nfs/templates/csi-nfs-controller.yaml

```diff
@@ -15,6 +16,16 @@ spec:
 {{ include "nfs.labels" . | indent 6 }}
         app: csi-nfs-controller
     spec:
+      affinity:
```

Comment:
should also update tgz file, check the guidance in ut failure, thanks.

Reply:
thanks, I'll update it, as well